Packs/OpenAI/Integrations/OpenAIGPT/OpenAIGPT.yml

```diff
@@ -0,0 +1,201 @@
+category: Messaging and Conferencing
+name: OpenAIGPT
+display: OpenAI GPT
+description: Designed to assist security professionals with security investigations, threat hunting, anomaly detection. Leveraging OpenAI GPT models' natural language conversational capabilities.
+commonfields:
+  id: OpenAIGPT
+  version: -1
+
+configuration:
+- display: Server URL
+  name: url
+  required: false
```

Comment:
server URL is not required ? can the integration run without it ?

Reply:
right, made it required.